### PR TITLE
Update Docs

### PR DESCRIPTION
**Pull request by KarthikRIyer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ run_examples.sh
 ```
 Jupyter Notebook examples are under the `Notebooks` directory.
 
-The resultant images are stored in the `examples/Reference` directory. The images rendered by each of the backends are stored their respective directories: [agg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/agg) and [svg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/svg)
+The resultant images are stored in the `examples/Reference` directory. The images rendered by each of the backends are stored their respective directories: [agg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/agg), [svg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/svg) and [quartz](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/svg)
```
Comment (BradLarson) [inline README.md]:
```suggestion
The resultant images are stored in the `examples/Reference` directory. The images rendered by each of the backends are stored their respective directories: [agg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/agg), [svg](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/svg) and [quartz](https://github.com/KarthikRIyer/swiftplot/tree/master/examples/Reference/quartz)
```

**Comment by BradLarson** [comment]
This looks good to me, aside from the one small link change above. Once the Core Graphics renderer is ready to merge in, I'd say you're safe to pull this in at that time.

**Comment by KarthikRIyer** [comment]
@BradLarson is it fine if I remove the CoreGraphics part and pull this in, just to have a working and documented submission?

**Comment by BradLarson** [comment]
Sure, that makes sense.

Regarding the Jupyter FreeType issues, is there a way to have a short-term workaround where those are temporarily deactivated on Linux platforms, just until this is resolved? So you can still see something in the notebook.

**Comment by KarthikRIyer** [comment]
I think we should be able to do that. I'll give it a try tomorrow. If that doesn't work I'll disable FreeType for now and use the old method instead.

**Pull request merged**